vocabularies/Org.OData.Temporal.V1.xml

```diff
@@ -136,30 +147,17 @@
       <!-- Operations on Temporal Objects -->
 
       <Action Name="Update" IsBound="true" EntitySetPath="timeslices">
-        <Annotation Term="Core.Description" String="Updates all time slices of a temporal object in a given period of application time after splitting partially overlapping time slices" />
+        <Annotation Term="Core.Description" String="Adds time slices to a time-slice entity set, splitting other time slices in case of overlap" />

```

Comment:
Some of the splitted time slices are updated, which was part of the original text.

Let's first try to describe this in the specification, then copy parts of the text here.